velog/velog.html

```diff
@@ -0,0 +1,269 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="velog.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Do+Hyeon&family=Gemunu+Libre:wght@600&family=IBM+Plex+Sans+KR:wght@300;500&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/06176f9978.js" crossorigin="anonymous"></script>
+    <title>Bonglog</title>
+</head>
+<body>
+    <header>
+        <div class = "header__logo">
+            <p>Bonglog</p>
+        </div>
```

Comment:
요건 굳이 div로 묶지 않아도 될 것 같아요
```suggestion
            <p class="header__logo">Bonglog</p>
```